privacy-contracts/src/main/solidity/DefaultOnChainPrivacyGroupManagementContract.sol

```diff
@@ -3,6 +3,7 @@ import "./OnChainPrivacyGroupManagementInterface.sol";
 
 contract DefaultOnChainPrivacyGroupManagementContract is OnChainPrivacyGroupManagementInterface {
 
+    address private _owner;
```

Comment:
Just to confirm. We are setting the owner when we inject the contract into the newly created private state, correct?

Reply:
The owner is set when the addParticipants is called the first time:
   function addParticipants(bytes32[] memory _accounts) public returns (bool) {
        require(!_canExecute);
        if (_owner == address(0x0)) {
            _owner = tx.origin;
        }
...